### ISSUE TITLE
Cannot download song with forward slash in title

### ISSUE THREAD

**Issue by BobDaGecko** [deepjyoti30/ytmdl#139]
# Bug report

## Checklist

 - [x] I have searched for similar bugs in the issues.
 - [x] I will add the bug label to this issue.

## Version Info

- Ytmdl version: `2020.12.02-dev`
- Youtube-dl version: `2020.12.31`

## How Ytmdl was installed?

pip

## Console Log

```console
bobdagecko@P5SQ8Q8:/home/bobdagecko/music# ytmdl https://www.youtube.com/watch?v=EqgDCi5Cwh0 --skip-meta --level DEBUG
 ==> Logger running in DEBUG mode
 ==> Passed args: Namespace(SONG_NAME=['https://www.youtube.com/watch?v=EqgDCi5Cwh0'], album=None, artist=None, choice=None, disable_file=False, disable_metaadd=False, disable_sort=False, format='mp3', get_opts=False, ignore_errors=False, keep_chapter_name=False, level='DEBUG', list=None, list_level=False, manual_meta=False, nolocal=False, pl_end=None, pl_items=None, pl_start=None, proxy=None, quiet=False, skip_meta=True, song=None, title_as_name=False, trim=False, url=None)
 ==> ['https://www.youtube.com/watch?v=EqgDCi5Cwh0']
 ==> https://www.youtube.com/watch?v=EqgDCi5Cwh0
 ==> Searching to see if already present in /home/bobdagecko/music
 ==> proxy passed: None
 ==> Passed format: mp3
 ==> Searching Youtube for https://www.youtube.com/watch?v=EqgDCi5Cwh0
 ==> Choose One (One with [M] is verified music)
 [1] HITMAN 2 Soundtrack - Hawke's Bay (Custom Mix) by Salem with dur 25:44
 [2] Scoutman 2 [#Scoutvideogameadventure] by WAMCrash with dur 2:33
 [3] Hitman 2 - Ballistic 47: Server Farm (SA) by Nax with dur 4:39
 [4] This is Pretty Much Abience (Springtrap and Deliah: The Sequel) by erdinoia [CHAPTER 1] by Finnshuntaro with dur 5:58
 ==> Enter Choice (1): 1
 ==> Downloading HITMAN 2 Soundtrack - Hawke's Bay (Custom Mix) in 320kbps
 ==> Saving the files to: /root/.cache/ytmdl
 ==> /root/.cache/ytmdl/HITMAN#2#Soundtrack#-#Hawke's#Bay#(Custom#Mix).mp3
24 MB  | 40 Mb/s || ETA: 5 s                                                                                                                                   |▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓| 100% ==> Downloaded!
 ==> The song has chapters in it. Each part will be extracted and worked accordingly.
 ==> [{'start_time': 0.0, 'end_time': 120.0, 'title': 'Introduction'}, {'start_time': 120.0, 'end_time': 347.0, 'title': 'Investigating'}, {'start_time': 347.0, 'end_time': 421.0, 'title': 'Multiple Hostiles'}, {'start_time': 421.0, 'end_time': 558.0, 'title': 'Stealth'}, {'start_time': 558.0, 'end_time': 680.0, 'title': 'Trespassing'}, {'start_time': 680.0, 'end_time': 802.0, 'title': 'Illegal Activity'}, {'start_time': 802.0, 'end_time': 959.0, 'title': 'Suspicious/Arrested'}, {'start_time': 959.0, 'end_time': 1177.0, 'title': 'Combat + Hunted'}, {'start_time': 1177.0, 'end_time': 1267.0, 'title': 'Following Mission Story'}, {'start_time': 1267.0, 'end_time': 1277.0, 'title': 'Target Eliminated'}, {'start_time': 1277.0, 'end_time': 1511.0, 'title': 'Extraction'}, {'start_time': 1511.0, 'end_time': 1544, 'title': 'Successful Escape'}]
 ==> Has the attribute
 ==> Current extracted title for the song is: `Introduction`
 ==> Most extracted titles are not accurate and they affect the meta search
Would you like to change [y/n] (y): n
 ==> song_name:  Introduction  song_meta:  Introduction
 ==> 0.0:120.0
 ==> Skip Meta: True
 ==> ext: mp3
 ==> Moving to: /home/bobdagecko/music
 ==> Moved to /home/bobdagecko/music...
 ==> Done
 ==> Has the attribute
 ==> Current extracted title for the song is: `Investigating`
 ==> Most extracted titles are not accurate and they affect the meta search
Would you like to change [y/n] (y): n
 ==> song_name:  Investigating  song_meta:  Investigating
 ==> 120.0:347.0
 ==> Skip Meta: True
 ==> ext: mp3
 ==> Moving to: /home/bobdagecko/music
 ==> Moved to /home/bobdagecko/music...
 ==> Done
 ==> Has the attribute
 ==> Current extracted title for the song is: `Multiple Hostiles`
 ==> Most extracted titles are not accurate and they affect the meta search
Would you like to change [y/n] (y): n
 ==> song_name:  Multiple Hostiles  song_meta:  Multiple Hostiles
 ==> 347.0:421.0
 ==> Skip Meta: True
 ==> ext: mp3
 ==> Moving to: /home/bobdagecko/music
 ==> Moved to /home/bobdagecko/music...
 ==> Done
 ==> Has the attribute
 ==> Current extracted title for the song is: `Stealth`
 ==> Most extracted titles are not accurate and they affect the meta search
Would you like to change [y/n] (y): n
 ==> song_name:  Stealth  song_meta:  Stealth
 ==> 421.0:558.0
 ==> Skip Meta: True
 ==> ext: mp3
 ==> Moving to: /home/bobdagecko/music
 ==> Moved to /home/bobdagecko/music...
 ==> Done
 ==> Has the attribute
 ==> Current extracted title for the song is: `Trespassing`
 ==> Most extracted titles are not accurate and they affect the meta search
Would you like to change [y/n] (y): n
 ==> song_name:  Trespassing  song_meta:  Trespassing
 ==> 558.0:680.0
 ==> Skip Meta: True
 ==> ext: mp3
 ==> Moving to: /home/bobdagecko/music
 ==> Moved to /home/bobdagecko/music...
 ==> Done
 ==> Has the attribute
 ==> Current extracted title for the song is: `Illegal Activity`
 ==> Most extracted titles are not accurate and they affect the meta search
Would you like to change [y/n] (y): n
 ==> song_name:  Illegal Activity  song_meta:  Illegal Activity
 ==> 680.0:802.0
 ==> Skip Meta: True
 ==> ext: mp3
 ==> Moving to: /home/bobdagecko/music
 ==> Moved to /home/bobdagecko/music...
 ==> Done
 ==> Has the attribute
 ==> Current extracted title for the song is: `Suspicious/Arrested`
 ==> Most extracted titles are not accurate and they affect the meta search
Would you like to change [y/n] (y): n
 ==> song_name:  Suspicious/Arrested  song_meta:  Suspicious/Arrested
 ==> 802.0:959.0
 ==> Skip Meta: True
 ==> ext: mp3
 ==> Moving to: /home/bobdagecko/music
 ==> [Errno 2] No such file or directory: '/home/bobdagecko/music/Suspicious/Arrested.mp3'
```

## Description

When a song's name has a forward slash it messes with the file's path and breaks the program. The `--ignore-errors` flag does not fix this.

**Comment by deepjyoti30** [comment]
@BobDaGecko Thanks for reporting the issue. The issue should be fixed in 86cea072e64b4215c87eba49a868ff83be4b25f9

I think the bug here was that `ytmdl` was not able to replace special characters in the string automatically. However, I will take a look at why `--ignore-errors` was not working.

**Comment by BobDaGecko** [comment]
@deepjyoti30 
Couple things. The first is I think your fix caused the movement of files not to work. The file is renamed to the path plus the song name. So I would get -home-user-music-<song_name>.mp3 in the current directory I run ytmdl in.

Also I clearly primarily use ytmdl for downloading videos with timestamps. I am not as interested in the meta data so I was wondering if you could enhance the --quiet flag to avoid asking if I want to change the title for each timestamp. I can make this a feature request if you need me too. Thanks for the quick fix though.

**Comment by deepjyoti30** [comment]
@BobDaGecko Sure I will fix that issue.

Also, do consider raising a proper feature request with a proper explanation of what exactly you want. I will consider it for sure!

**Comment by deepjyoti30** [comment]
@BobDaGecko Should be fixed in 326fa8ab598add2f9afb6c245a57febeb38f8b67
